app/controllers/books_controller.rb

```diff
@@ -5,7 +5,7 @@ class BooksController < ApplicationController
 
   # GET /books or /books.json
   def index
-    @books = Book.all
+    @books = Book.page(params[:page]).order(:id).per(3)
```

Comment:
```suggestion
    @books = Book.order(:id).page(params[:page]).per(3)
```

慣習的なものですが、orderの次にkaminari用のメソッドを持ってくることが多いですね

Reply:
orderの位置をkaminariの前に移動しました。
`@books = Book.order(:id).page(params[:page]).per(3)`